indico/modules/attachments/models/attachments.py

```diff
@@ -240,10 +240,11 @@ def get_download_url(self, absolute=False):
         :param absolute: If the returned URL should be absolute.
         """
         if ContextManager.get('offlineMode'):
-            return _offline_download_url(self)
+            url = _offline_download_url(self)
         else:
             filename = self.file.filename if self.type == AttachmentType.file else 'go'
-            return url_for('attachments.download', self, filename=filename, _external=absolute)
+            url = url_for('attachments.download', self, filename=filename, _external=absolute)
+        return url
```

Comment:
I added something there at one point and forgot to restore the previous version.